src/interface/report/BossEventLoader.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { Trans } from '@lingui/macro';
+import ExtendableError from 'es6-error';
+
+import { fetchEvents, LogNotFoundError } from 'common/fetchWclApi';
+import { captureException } from 'common/errorLogger';
+import makeAnalyzerUrl from 'interface/common/makeAnalyzerUrl';
+import { findByBossId } from 'raids';
+
+import Ghuun from './images/Ghuun.gif';
+import handleApiError from './handleApiError';
+import './EventParser.css';
+
+export class BossEventsLoadError extends ExtendableError {
+  reason = null;
+  constructor(reason) {
+    super();
+    this.reason = reason;
+  }
+}
+
+class BossEventsLoader extends React.PureComponent {
```

Comment:
```suggestion
class BossEventLoader extends React.PureComponent {
```

Please try to keep the class name the same as its file